src/Aspire.Hosting/ApplicationModel/ResourceNotificationService.cs

```diff
@@ -709,6 +714,23 @@ private sealed class ResourceNotificationState
         public long GetNextVersion() => _lastVersion++;
         public CustomResourceSnapshot? LastSnapshot { get; set; }
     }
+
+    internal static bool IsMicrosoftOpenType(Type type)
+    {
+        var microsoftOpenPublicKey = new byte[]
```

Comment:
We should make this a static ReadOnlySpan so we don't create a new byte array every time this method is called.